token/cli/src/bench.rs

```diff
@@ -266,15 +266,15 @@ fn get_token_addresses_with_seed(
         .collect()
 }
 
-async fn is_valid_token(rpc_client: &RpcClient, token: &Pubkey) -> Result<(), Error> {
+async fn is_valid_token(rpc_client: &RpcClient, token: &Pubkey) -> Result<Pubkey, Error> {
```

Comment:
Now that this method returns data, should we rename to be more self-descriptive? (I realize this is only used in `bench.rs`, so not a huge deal)

Reply:
You're totally right, I punted because I couldn't come up with a good name. Best I got is `get_program_id_from_valid_mint`  or `get_valid_mint_program_id` -- you?

Reply:
I like `get_valid_mint_program_id` well enough :)